docs/systemd.md

```diff
@@ -0,0 +1,77 @@
+## systemd cgroup driver
+
+By default, runc creates cgroups and sets cgroup limits on its own (this mode
+is known as "fs" cgroup driver). When `--systemd-cgroup` global option is given
+(as in e.g. `runc --systemd-cgroup run ...`), runc switches to "systemd" cgroup
+driver.
+
+In particular, on `run` or `create`, runc requests systemd to create a scope
+(a.k.a. transient unit), usually named like `runc-$ID.scope`, and puts
+container task into that scope. The resource limit of the systemd scope created
+are set by runc by translating the runtime spec resources to systemd unit
+properties. If an older systemd version (which does not support some resources)
+is used, runc do not set those resources.
+
+As there are some cgroup properties that can not be set via systemd, runc
+systemd cgroup driver is backed by fs driver (in other words, cgroup limits are
+set via both systemd unit properties AND by writing to cgroupfs files).
+
+The set of runtime spec resources which is translated by runc to systemd unit
+properties depends on cgroup version being used.
+
+#### cgroup v1
+
+| runtime spec resource | systemd property name | min systemd version |
+|-----------------------|-----------------------|---------------------|
+| memory.limit          | MemoryLimit           | any                 |
```

Comment:
nit: "any" is not true because probably it doesn't work with systemd v1 😛 
Maybe better to just leave these cells blank.

Reply:
fixed